TeslaSUI/TeslaSUI/Views/ClimateView.swift

```diff
@@ -0,0 +1,393 @@
+                ))
+                .overlay {
+                    Circle()
+                        .stroke(
+                            LinearGradient(colors: [.topGradient, .bottomGradient],
+                                           startPoint: .topLeading,
+                                           endPoint: .bottomTrailing), lineWidth: 2
+                        )
+                }
+                .neumorphismUnSelectedStyle()
+        }
+        .padding(.top, -20)
+        
+    }
+    
+    private var bottomSheetCapsuleView: some View {
+        Capsule()
+            .fill(.gray)
+            .frame(width: 80, height: 3)
+            .padding(.top)
+    }
+    
+    private var bottomSheetTextView: some View {
+        VStack(alignment: .leading) {
+            Text(Constants.Text.acIsOn)
+                .font(.system(size: 20, weight: .semibold))
+            Text(Constants.Text.turnOnButtonText)
+                .font(.system(size: 18))
+                .frame(width: 250, height: 50)
+                .lineLimit(2)
+                .foregroundColor(.gray)
+                .padding(.top, -10)
+        }
+        .padding(.leading, 30)
+    }
+    
+    private var progressViewGradient = LinearGradient(
+        colors: [.progressViewBottomGradient, .progressViewTopGradient],
+        startPoint: .leading,
+        endPoint: .trailing)
+    
+    private var climateSettingsView: some View {
+        DisclosureGroup(isExpanded: $climateViewModel.settings) {
+            VStack {
+                acClimateSettingView
+                fanClimateSettingView
+                heatClimateSettingView
+                autoClimateSettingView
+            }
+        } label: {
+            HStack {
+                Text(Constants.Text.climateSettings)
+                    .font(.system(size: 17, weight: .semibold))
+                    .foregroundColor(.white)
+            }
+        }
+        .accentColor(.white)
+        .padding()
+    }
+    
+    private var acClimateSettingView: some View {
+        HStack {
+            Text(Constants.Text.ac)
+                .font(.system(size: 17, weight: .semibold))
+                .frame(width: 50)
+            Button {
+                climateViewModel.isAcButtonTapped.toggle()
+            } label: {
+                Image(Constants.Images.snowIconImageName)
+                    .resizable()
+                    .modifier(ButtonImageModifire())
+            }
+            .neumorphismUnSelectedCircleStyle()
+            .overlay(
+            Circle()
+                .stroke(gradient, lineWidth: 2)
+                .opacity(climateViewModel.isAcButtonTapped ? 1 : 0)
+            )
+            
+            Slider(value: $climateViewModel.circleProgress,
+                   in: Constants.Integer.sliderRange,
+                   step: 1)
+            .frame(width: 220)
+        }
+    }
+    
+    private var fanClimateSettingView: some View {
```

Comment:
```suggestion
    private var fanClimateSettingView: some View {
```
много одинакового когда, можно было через форич сделать эти 4 вью
это на твое усмотрение) можно не править